diff --git a/lib/xdg/com.ayugram.desktop.desktop b/lib/xdg/com.ayugram.desktop.desktop
index d9a12b7532..9f66f99498 100644
--- a/lib/xdg/com.ayugram.desktop.desktop
+++ b/lib/xdg/com.ayugram.desktop.desktop
@@ -2,7 +2,7 @@
 Name=AyuGram Desktop
 Comment=Desktop version of AyuGram - ToS breaking Telegram client
 TryExec=ayugram-desktop
-Exec=env DESKTOPINTEGRATION=1 ayugram-desktop -- %u
+Exec=ayugram-desktop -- %u
 Icon=com.ayugram.desktop
 Terminal=false
 StartupWMClass=AyuGram
